plone/app/linkintegrity/browser/update.py

```diff
@@ -69,8 +50,6 @@ def update(self):
         catalog = getToolByName(self.context, 'portal_catalog')
         count = 0
         query = {}
-        if HAS_MULTILINGUAL and 'Language' in catalog.indexes():
-            query['Language'] = 'all'
 
         for brain in catalog(query):
```

Comment:
@pbauer 
- The query variable is now useless, isn't it? ``catalog()`` would also do it, no?
- Wasn't the catalog called wrong? It should rather read ``catalog(**query)``. 
- I recognize that the catalog query only returns items, which the user is allowed to see. At least we should call the catalog with a query like ``catalog(show_inactive=True)``. Still, the ``kw['allowedRolesAndUsers'] = self._listAllowedRolesAndUsers(user)`` query is added. See: https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/CatalogTool.py#L437 (I guess, we're using this catalog tool, don't we?)

Reply:
Mostly true. The view requires cmf.ManagePortal anyway, so it will find all content.
I'll remove the query.